std/machines/large_field/arith256_memory.asm

```diff
@@ -32,7 +32,8 @@ machine Arith256Memory(mem: Memory) with
     let operation_selectors = [is_affine, is_mod, is_ec_add, is_ec_double];
     array::map(operation_selectors, |s| force_bool(s));
     array::map(operation_selectors, fixed_inside_32_block);
-    let operation_id = sum(4, |i| 2 ** i * operation_selectors[i]);
+    col witness operation_id;
```

Comment:
I had to change this because if I understand correctly, the jit prover function heuristic expects an algebraic expression in the condition of `compute_from_multi_if`, but this is an `expr`

Reply:
If I use `col operation_id: inter`, then I get a [panic in the optimizer](https://github.com/powdr-labs/powdr/blob/2614c0402de171eaf2c6a4cc1967af0cedda64c2/pilopt/src/lib.rs#L132-L133) with `n.type_args = Some([])`